migrations/transformWordpressToUrlSlug.js

```diff
@@ -8,7 +8,7 @@ module.exports = function (migration) {
     from: ['wordpress_url', 'urlSlug'],
     to: ['urlSlug'],
     transformEntryForLocale: function (fromFields, currentLocale) {
-      const wordPressUrl = fromFields.wordpressUrl;
+      const wordPressUrl = fromFields.wordpress_url;
```

Comment:
👍 I never know with Contentful…I think _old_ are definitely all `snake_case`, whereas if you create a new one, it defaults to `camelCase`..?  I think it even gets parsed by `middleman-contentful` 🤷